bigquery_etl/schema/ads_derived.yaml

```diff
@@ -0,0 +1,352 @@
+fields:
```

Comment:
This YAML schemas are migrated as-is from [private_bqetl](https://github.com/mozilla/private-bigquery-etl/tree/main/canonical) so they can be used with the command.